src/tutorials/0004-mutable-file-system/02.js

```diff
@@ -8,7 +8,12 @@ const validate = async (result, ipfs) => {
   } else if (!!result && result.cid.toString() === 'QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn') {
     return {
       success: utils.validationMessages.SUCCESS,
-      logDesc: "Here's the status of your root directory ( `/` ). Notice that it has a CID even though it doesn't have contents yet. Every empty IPFS node has this exact same CID, because their non-existent contents are identical!",
+      logDesc: [
+        "Here's the status of your root directory (`/`).",
```

Comment:
```suggestion
        "Here's the status of your root directory (`/`). The output is very long because the CID is represented as an `Object` internally, but if you scroll down we'll offer you a more condensed view.",
```